src/edu/ucsd/library/xdre/statistic/analyzer/LogAnalyzer.java

```diff
@@ -90,11 +90,17 @@ public void analyze(File logFile) throws IOException{
 					if (line.indexOf("\"-\"", spIdx) > 0 || line.indexOf("archive.org_bot", spIdx) > 0)
 						continue;
 
-					// ignore hits from MSCL collection pages: http://library.ucsd.edu/speccoll/
-					if (line.indexOf("http://library.ucsd.edu/speccoll/", spIdx) > 0)
-						continue;
-					
 					uri = getUri(line);
+					// Hits from MSCL collection pages: http://library.ucsd.edu/speccoll/
```

Comment:
Oh I was just referring to the code itself. Since it's targeted specifically for this use case, I thought perhaps extracting it to a separate method might make sense. Wasn't suggesting we change your overall solution.